pymongoexplain/__main__.py

```diff
@@ -48,6 +49,23 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
-        with open(file) as f:
-            exec(f.read())
+    parser = argparse.ArgumentParser(description="This script is intended to "
```

Comment:
You can define the helpstring as a multi-line string elsewhere in the file and then just it as a variable here. That would avoid this multi-line command.